packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx

```diff
@@ -249,6 +249,7 @@ const mapDispatchToProps = (dispatch: Dispatch, props: IFullProps) => {
 
       if (variables.user.signature) {
         delete variables.user.signature.name
+        delete variables.user.signature.__typename //to fix updating registrar bug
```

Comment:
Where is this field coming from? If we need to remove it here, do we need to fetch it from the backend in the first place?

Reply:
Thanks Riku. I spent much time today on thinking about it. This __typename field is automatically added from graphql when user is fetched. So I couldn't find out any easier way to remove it from backend and went for this aproach.

Reply:
when did you console log it? while updating registrar or creating registrar? __typename exists only at the time of updating registrar.

Reply:
![image](https://user-images.githubusercontent.com/57105414/168004725-3f5cb824-ebfa-44e8-8a32-78b871e20338.png)

Reply:
I got this in signature while updating registrar

Reply:
Thanks! Yea, it was just me trying to see it when creating the user. I'm just now thinking if the view is the best place to get rid of those extra fields. If we don't have a universal way of removing all __typenames from query results, then maybe it is... Ok, I'm happy with this 👍